llvm/include/llvm/SYCLLowerIR/ModuleSplitter.h

```diff
@@ -261,11 +261,13 @@ class ModuleSplitterBase {
 };
 
 SmallVector<ModuleDesc, 2> splitByESIMD(ModuleDesc &&MD,
-                                        bool EmitOnlyKernelsAsEntryPoints);
+                                        bool EmitOnlyKernelsAsEntryPoints,
+                                        bool ExcludeExternalFunctions);
 
 std::unique_ptr<ModuleSplitterBase>
 getDeviceCodeSplitter(ModuleDesc &&MD, IRSplitMode Mode, bool IROutputOnly,
-                      bool EmitOnlyKernelsAsEntryPoints);
+                      bool EmitOnlyKernelsAsEntryPoints,
+                      bool ExcludeExternalFunctions);
```

Comment:
Sorry for catching this late. Think this sounds better as ExcludeSYCLExternalFunctions?

Thanks